### PR TITLE
fix-vertical carousel arrows

### PR DESCRIPTION
**Pull request by udaypydi**
- [ ] `package.json:version` has been updated with `npm version patch` (or `major/minor` as appropriate)
- [ ] `@brainhubeu/react-carousel` version has been updated in `docs-www/package.json` to the same which is in `package.json:version`

### CODE REVIEW

**Comment by udaypydi** [comment]
@piotr-s-brainhub  this PR fixes the arrows. Need your help in sorting out the empty space in `dots` and `thumbnails`

File: src/components/Carousel.js [hunk 1]
```diff
@@ -546,12 +548,10 @@ class Carousel extends Component {
     const animationSpeed = this.getProp('animationSpeed');
     const transitionEnabled = this.state.transitionEnabled;
     const draggable = this.getProp('draggable') && children && children.length > 1;
-
```
Comment (piotr-s-brainhub) [inline src/components/Carousel.js:549]: could you revert these lines (an empty line where it was and no empty line where there was no empty line)?

Comment (piotr-s-brainhub) [reply src/components/Carousel.js:549]: thanks for reverting next lines but could you also revert empty line here?

Comment (udaypydi) [reply src/components/Carousel.js]: yeah done. I missed in the earlier commit.

**Pull request merged**